components/director/internal/domain/apptemplate/repository_test.go

```diff
@@ -20,7 +20,8 @@ import (
 func TestRepository_Create(t *testing.T) {
 	t.Run("Success", func(t *testing.T) {
 		// GIVEN
-		appTemplateModel := fixModelAppTemplate(testID, testName)
+		webhooksModel := fixModelApplicationWebhooks(testWebhookID, testID)
```

Comment:
We should come with a convention "App" or "Application" in fixture function names.
The difference I have in mind is "fixModelAppTemplate" and "fixModelApplicationWebhooks".